adal/src/androidTest/java/com/microsoft/aad/adal/AuthenticationContextTest.java

```diff
@@ -856,7 +856,7 @@ public void testRefreshTokenPositive() throws IOException, InterruptedException,
 
         final String response2 = "{\"id_token\":\""
                 + TEST_IDTOKEN
-                + "\",\"access_token\":\"TokenReturnsWithIdToken\",\"token_type\":\"Bearer\",\"expires_in\":\"10\",\"expires_on\":\"1368768616\",\"refresh_token\":\"refreshABC\",\"scope\":\"*\"}";
+                + "\",\"access_token\":\"TokenReturnsWithIdToken\",\"token_type\":\"Bearer\",\"expires_in\":\"3600\",\"expires_on\":\"1368768616\",\"refresh_token\":\"refreshABC\",\"scope\":\"*\"}";
```

Comment:
I don't really think it matters, this is a mocked data, if this simple test takes more than 10 seconds to finish, something must go wrong. Debugging would be a different story, but i don't think we should change data only for debugging purpose. 

---
In reply to: [70036751](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/676#discussion_r70036751) [](ancestors = 70036751)